service/src/main/resources/mybatis-config.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE configuration
+        PUBLIC "-//mybatis.org//DTD Config 3.0//EN"
+        "http://mybatis.org/dtd/mybatis-3-config.dtd">
+<configuration>
+    <settings>
+        <setting name="mapUnderscoreToCamelCase" value="true" />
+    </settings>
+    <environments default="development">
+        <environment id="development">
+            <transactionManager type="JDBC"/>
+            <dataSource type="org.apache.celeborn.server.common.service.store.db.HikariDataSourceFactory">
+                <property name="jdbcUrl" value="${jdbcUrl}"/>
+                <property name="username" value="${username}"/>
+                <property name="password" value="${password}"/>
+                <property name="connectionTimeout" value="${connectionTimeout}"/>
+                <property name="idleTimeout" value="${idleTimeout}"/>
+                <property name="maxLifetime" value="${maxLifetime}"/>
+                <property name="maximumPoolSize" value="${maximumPoolSize}"/>
+            </dataSource>
+        </environment>
+    </environments>
+    <mappers>
+        <package name="org.apache.celeborn.server.common.service.store.db.mapper"/>
```

Comment:
can we list all classes instead? searching in classpath with package name may consume certain memories, typically 500MiB+